utilities/db_backup/__main__.py

```diff
@@ -5,7 +5,7 @@
 from dataclasses import dataclass
 from datetime import datetime, timezone
 from pathlib import Path
-from typing import Dict, Generator, List, Optional
+from typing import Any, Dict, Generator, List, Optional
```

Comment:
Should we use this update to replace the `Dict` to`dict` in those files? it seems like there is both in this one

Reply:
I don't know if it's directly related to this change. We could activate https://docs.astral.sh/ruff/rules/#flake8-future-annotations-fa and have the linter flag warnings for this so that it's easier to track, then it would be flagged if we have a file that should do an `from __future__ import annotations` and we'd get warnings in all of the places where we're using the old format. Though I'd prefer to have that as separate PRs, or even the change from Dict -> dict if we don't enable the new rule.